server_installation/topics/operating-mode/crossdc/proc-setting-up-infinispan.adoc

```diff
@@ -0,0 +1,160 @@
+[id='setting-up-infinispan-{context}']
+= Setting Up {jdgserver_name} {jdgserver_version_latest} Clusters
+You begin this procedure by creating remote {jdgserver_name} clusters that can back up {project_name} data.
```

Comment:
This sounds a little odd to me. This procedure is about creating remote RHDG clusters. 

Do you mean that you begin the process of setting up xsite replication by creating remote RHDG clusters?

Reply:
Yes. So how about 
You begin the process of cross-site replication by creating remote {jdgserver_name} clusters that can back up {project_name} data.

Reply:
Create clusters of {jdgserver_name} that store and back up {project_name} data across data centers.

Not sure about "the process of cross-site replication" tbh.